tests/test_coils.py

```diff
@@ -372,7 +492,7 @@ def test_dunder_methods(self):
         del coils1[-2]
         assert len(coils1) == 4
         assert coils1[-1] is coil2
-        assert coils1[-2][0].__class__ is coil1.__class__
+        assert coils1[-2].__class__ is coil1.__class__
```

Comment:
why did this change?

Reply:
because before `CoilSet.from_symmetry` was making a `CoilSet` with each member another `CoilSet` for each field period, and now it is making a `CoilSet` where each member is a `Coil`